pkg/credentialprovider/aws/aws_credentials.go

```diff
@@ -34,6 +34,7 @@ import (
 // and credentialprovider.
 var AWSRegions = [...]string{
 	"us-east-1",
+	"us-east-2",
```

Comment:
Might not have existed when this list was aggregated. @justinsb PTAL. This is unrelated to the PR and should probably be left out.